src/smoothy.rs

```diff
@@ -0,0 +1,57 @@
+// Disclaimer:
+// - Beyond an educated guess that samples might denote an amplitude between -1 to 1, I have no idea how any of this works.
+// - I have not listened to what this produces. The initial idea was to make things sluggish by limiting how much samples would differ from the previous ones. But perhaps all this does is to reduce volume in a slightly random way? Only one way to find out: submitting a PR and having you describe the effect to me.
```

Comment:
I found with lower "blend" values it sounds like a high pass filter, once the blend values get higher then it brings the volume down.